test/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ macro(onedpl_add_test test_source_file)
     set_target_properties(${_test_name} PROPERTIES CXX_EXTENSIONS NO)
 
     add_test(NAME ${_test_name} COMMAND ${CMAKE_CURRENT_BINARY_DIR}/${_test_name})
+    set_tests_properties(${_test_name} PROPERTIES SKIP_RETURN_CODE 77)
```

Comment:
Not sure what it is

Reply:
I will erase this change in the next commit.